i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: /tim-features
+title: TIM Features
+hide_title: true
+---
+
+import DocCardList from '@theme/DocCardList';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">TIM Features</h2>
+  </div>
+  
+</div>
+<br></br>
+
+:::note
+
+Sie werden nur Zugriff auf die TIM Features haben, wenn Ihr Account auf einem TIM Server ist. Möglicherweise sehen Sie diese Features nicht.
```

Comment:
Ich würde sagen 'Die TIM Features werden nur für Sie verfügbar sein, wenn..' Aber der zweite Satz ist auf Deutsch viel logischer so, vielleicht war es vorhin ein Übersetzungsproblem.

Reply:
Der Satz sagt exakt genau das selbe aus wie auf Englisch. Ich werde ihn einfach komplett streichen in beiden Versionen